### PR TITLE
Enable notifs for HMS HTAN bucket

### PR DESCRIPTION
**Pull request by clarisse-lau**
- add HTAN collaborator to HMS bucket
- set up HMS bucket with s3-synapse-sync lambda

### CODE REVIEW

File: config/prod/s3-synapse-sync.yaml [hunk 1]
```diff
@@ -17,7 +17,8 @@ parameters:
       "htan-dcc-tma-tnp":{"SynapseProjectId":"syn22041595"},
       "htan-dcc-msk":{"SynapseProjectId":"syn23448901"},
       "htan-dcc-duke":{"SynapseProjectId":"syn23511961"},
-      "htan-dcc-tnp-sardana":{"SynapseProjectId":"syn24984270"}
+      "htan-dcc-tnp-sardana":{"SynapseProjectId":"syn24984270"},
+      "htan-dcc-hms":{"SynapseProjectId":"syn22123910"}
```
Comment (BrunoGrandePhD) [inline config/prod/s3-synapse-sync.yaml:21]: @clarisse-lau: How did this bucket work before this?

**Review by BrunoGrandePhD — APPROVED** [review]
🚀

File: config/prod/s3-synapse-sync.yaml (hunk 1 above)
Comment (clarisse-lau) [reply config/prod/s3-synapse-sync.yaml:21]: Xengie and I had set up S3 buckets for each center and attached them to their respective Synapse projects. For some centers e.g. Vanderbilt and CHOP that are uploading data on Synapse, these buckets are just used as Synapse storage locations. When/if collaborators request access to upload directly to a bucket, then we set it up with the s3-synapse-sync lambda

Comment (BrunoGrandePhD) [reply config/prod/s3-synapse-sync.yaml:21]: Ah, I see! Thanks for the explanation!

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**